### PR TITLE
Add Validator247 Services

### PR DESCRIPTION
**Pull request by Validator247**

### CODE REVIEW

File: testnets/prysmdevnet/chain.json [hunk 1]
```diff
@@ -115,6 +120,9 @@
         "address": "https://prysm-rpc.validatorvn.com/",
         "provider": "ValidatorVN"
       },
+      {
+        "address": "https://prysm-testnet-rpc.validator247.com/",
+        "provider": "Validator247"
```
Comment (JeremyParish69) [inline testnets/prysmdevnet/chain.json:125]: this addition is missing a closing bracket. Please add new additions to the end of their respective arrays.

Comment (Validator247) [reply testnets/prysmdevnet/chain.json:125]: I added it according to the format, sorry for the mistake

Comment (JeremyParish69) [reply testnets/prysmdevnet/chain.json:125]: And Please add new additions to the end of their respective arrays.

Comment (Validator247) [reply testnets/prysmdevnet/chain.json:130]: added rpc/api/grpc/seed/persistent_peers...

**Pull request merged**